webapp/src/shared/components/Button/Button.css

```diff
@@ -1,6 +1,6 @@
 .button {
-  width: 312px;
-  height: 58px;
+  width: 15rem;
+  height: 3rem;
```

Comment:
Yes, you are right. I was tweaking around sizes because this buttons (with Figma sizes) where too big for desktop. So I adjust them a little bit so they look nice on mobile on desktop. I will update Figma library too :ok_hand: